x-pack/plugins/actions/server/integration_tests/connector_types.test.ts

```diff
@@ -50,11 +53,34 @@ describe('Connector type config checks', () => {
 
   for (const connectorTypeId of connectorTypes) {
     test(`detect connector type changes for: ${connectorTypeId}`, async () => {
-      const connectorType = actionTypeRegistry.get(connectorTypeId);
+      const {
+        getService,
+        validate: { config, params, secrets },
+      } = actionTypeRegistry.get(connectorTypeId);
 
-      expect(connectorType?.validate.config.schema.getSchema!().describe()).toMatchSnapshot();
-      expect(connectorType.validate.secrets.schema.getSchema!().describe()).toMatchSnapshot();
-      expect(connectorType.validate.params.schema.getSchema!().describe()).toMatchSnapshot();
+      // SubActionConnector
+      if (getService) {
+        const subActions = getService({
+          config: {},
+          configurationUtilities: actionsConfigMock.create(),
+          connector: { id: 'foo', type: 'bar' },
+          logger: loggerMock.create(),
+          secrets: {},
+          services: {} as Services,
+        }).getSubActions();
+
+        subActions.forEach((subAction) => {
+          // @ts-ignore
```

Comment:
I'm seeing this error too, but weirdly enough only in the typecheck script, not in the IDE (the `getSchema` method is indeed defined in the `Type` class).

nit: maybe explaining this in a `// @ts-expect-error` would be useful for future reference 🙂

Reply:
My IDE complains about  `// @ts-expect-error` as there is no error for it :)